pvm/ts/src/utils/parsers.ts

```
@@ -1,25 +1,59 @@
     const index = parseInt(params[1].trim());
 
+    if (params.length < 2) {
+        {
+            return {
+                from: tx.from,
+                to: tx.to,
+                value: tx.value,
+                type: action,
+                index: index
+            };
+        }
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)